app-doc/heirloom-doctools/files/999999.config

```diff
@@ -0,0 +1,24 @@
+INSTALL=/usr/bin/install
```

Comment:
Is there any difference between the two .config files? If no, then please use one file, and refer to it explicitly in both ebuilds.